package.json

```diff
@@ -79,15 +69,14 @@
     "Ramon Navarro Bosch <r.navarro@iskra.cat>"
   ],
   "dependencies": {
-    "@angular/common": "2.0.0-rc.1",
-    "@angular/compiler": "2.0.0-rc.1",
-    "@angular/core": "2.0.0-rc.1",
-    "@angular/http": "2.0.0-rc.1",
-    "@angular/platform-browser": "2.0.0-rc.1",
-    "@angular/platform-browser-dynamic": "2.0.0-rc.1",
-    "@angular/platform-server": "2.0.0-rc.1",
-    "@angular/router": "2.0.0-rc.1",
-    "angular2-universal": "~0.101.4",
+    "@angular/common": "2.0.0-rc.2",
+    "@angular/compiler": "2.0.0-rc.2",
+    "@angular/core": "2.0.0-rc.2",
+    "@angular/http": "2.0.0-rc.2",
+    "@angular/platform-browser": "2.0.0-rc.2",
+    "@angular/platform-browser-dynamic": "2.0.0-rc.2",
+    "@angular/platform-server": "2.0.0-rc.2",
+    "@angular/router": "3.0.0-alpha.6",
     "body-parser": "^1.15.0",
```

Comment:
@ebrehault @bloodbare I'm wondering if we should specifically pin (no "^" or "~") all versions right from the start to have a more stable CI build. What do you usually do in your JS projects?